js/passwordManager/bitwarden.js

```diff
@@ -161,41 +153,43 @@ class Bitwarden {
       console.warn(e)
     }
 
-    // show credentials dialog
-
-    var signInFields = [
+    // show ask-for-credential dialog
+    const signInFields = [
+      { placeholder: 'Server URL (Leave blank for the default Bitwarden server)', id: 'url', type: 'text' },
       { placeholder: 'Client ID', id: 'clientID', type: 'password' },
       { placeholder: 'Client Secret', id: 'clientSecret', type: 'password' }
     ]
 
-    const credentials = ipcRenderer.sendSync('prompt', {
-      text: l('passwordManagerBitwardenSignIn'),
-      values: signInFields,
-      ok: l('dialogConfirmButton'),
-      cancel: l('dialogSkipButton'),
-      width: 500,
-      height: 260
-    })
-
-    for (const key in credentials) {
-      if (credentials[key] === '') {
-        throw new Error('no credentials entered')
+    const credentials = ipcRenderer.sendSync(
+      'prompt',
+      {
+        text: l('passwordManagerBitwardenSignIn'),
+        values: signInFields,
+        ok: l('dialogConfirmButton'),
+        cancel: l('dialogSkipButton'),
+        width: 500,
+        height: 260
       }
+    )
+
+    if (credentials.clientID === '' || credentials.clientSecret === '') {
+      throw new Error('no credentials entered')
     }
 
-    const process = new ProcessSpawner(path, ['login', '--apikey'], {
-      BW_CLIENTID: credentials.clientID.trim(),
-      BW_CLIENTSECRET: credentials.clientSecret.trim()
-    })
+    credentials.url = credentials.url || 'bitwarden.com'
```

Comment:
It wasn't clear to me whether I was supposed to type "bitwarden.com" or "vault.bitwarden.com" - do they both work? If not, maybe we should special-case this so if the user enters one we convert to the other.